acceptance/acceptance_test.go

```diff
@@ -0,0 +1,87 @@
+// +build acceptance
+
+package acceptance
+
+import (
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/sclevine/spec"
+	"github.com/sclevine/spec/report"
+
+	h "github.com/buildpack/lifecycle/testhelpers"
+)
+
+func TestAcceptance(t *testing.T) {
+	spec.Run(t, "acceptance", testAcceptance, spec.Sequential(), spec.Report(report.Terminal{}))
```

Comment:
Drop `spec.Sequential()`, it's the default is only needed adjust the behavior in sub-Gs.